### DISCUSSION TITLE
Can not uninstall app

### DISCUSSION THREAD

**Discussion by chancezheng** [wixtoolset/issues#6770] [Announcements]
**Encountered at:** wixtoolset/repo@sha

**Detail:**
when i want to uninstall the app, but show the install view, the version is 3.11
![picture](https://user-images.githubusercontent.com/31839078/165752170-d21c4f14-569f-4246-8ac0-f78bdb6f1424.jpg)

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix
